### ISSUE TITLE
질문을 하는 과정에서 질문이 일부 누락되는 오류 발생

### ISSUE THREAD

**Issue by snsthqhr** [snsthqhr/Construction-defect-natural-language-processing-#7]
중간에 질문이 잘리는 오류가 발생

from report_excel import save_report_xlsx

MODEL_DIR = "./category_bert_model"
INDEX_DIR = "./retrieval_index"
MAX_LEN = 128
TOPK = 3

def mean_pooling(last_hidden, attention_mask):
    mask = attention_mask.unsqueeze(-1).float()
    summed = (last_hidden * mask).sum(dim=1)

**Comment by snsthqhr** [comment]
문제 발생 예시 질문 :신축아파트에 이번에 새롭게 이사를 오게 되었는데, 방에서 곰팡이 비슷한 냄새가 나고 벽쪽이 항상 젖어 있는거 같아요. 거실쪽 베란다에서는 물이 흐르는 자국이 보이고요. 베란다쪽 벽지도 누렇게 떠있고 페인트 칠도 마감이 잘 되지 않은거 같아요

**Comment by snsthqhr** [comment]
MAX_LEN 길이열의 문제임을 확인 하였음 256으로 크기 조정, 현재 모델로는 256이면 충분히 소화 할 수 있을거같음
